packaged_releases/docker/Dockerfile

```diff
@@ -8,6 +8,22 @@ FROM python:3.5.2-alpine
 ENV CLI_VERSION 0.1.9
 ENV CLI_DOWNLOAD_SHA256 4da806cc46acd8e634bc38eb81591b7ecf352703fb84d9b4cfcf4364237ac8b2
 
+# Metadata as defined at http://label-schema.org
+ARG VCS_REF
+ARG BUILD_DATE
+LABEL org.label-schema.schema-version="1.0" \
+      org.label-schema.vendor="Microsoft" \
+      org.label-schema.name="Azure CLI 2.0" \
+      org.label-schema.version=$CLI_VERSION \
+      org.label-schema.license="MIT" \
+      org.label-schema.description="The Azure CLI 2.0 is the new Azure CLI and is applicable when you use the Resource Manager deployment model." \
+      org.label-schema.url="https://docs.microsoft.com/en-us/cli/azure/overview" \
+      org.label-schema.usage="https://docs.microsoft.com/en-us/cli/azure/install-az-cli2#docker" \
+      org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vcs-url="https://github.com/Azure/azure-cli.git" \
+      org.label-schema.vcs-ref=$VCS_REF \
```

Comment:
As defined in http://label-schema.org/rc1/, `vcs-ref` is "Identifier for the version of the source code from which this image was built.".

The packaged release Docker image is not built from a single commit hash.
The source code is downloaded when running `docker build` from `https://azurecliprod.blob.core.windows.net/releases/azure-cli_packaged_${CLI_VERSION}.tar.gz`. This archive does not map to a single git commit.

Since all labels are optional, can we remove this one?